### ISSUE TITLE
Sensor status to unknown, Days_until to null

### ISSUE THREAD

**Issue by mdehaas** [pippyn/Home-Assistant-Sensor-Afvalbeheer#345]
I'm having issues which I noticed today because of the template content showing in my frontend. Updated to the latest release yesterday and the reboot was applied last night. The sensors do not seem to populate and there is an error in my log. I have not changed the config prior but tried troubleshooting this morning and only left some garbage stream enabled. The log contains the follwing:

>This error originated from a custom integration.
>
>Logger: homeassistant
>Source: custom_components/afvalbeheer/API.py:273
>Integration: Afvalbeheer (documentation)
>First occurred: 08:29:09 (1 occurrences)
>Last logged: 08:29:09
>
>Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/afvalbeheer/API.py", line 133, in async_update
    await self.collector.update()
  File "/config/custom_components/afvalbeheer/API.py", line 273, in update
    data = (response['ophaaldagen']['data'] + response['ophaaldagenNext']['data'])
KeyError: 'ophaaldagen'

The relevant portion of my config:

```yaml
afvalbeheer:
  wastecollector: MijnAfvalwijzer
  resources:
    - restafval
    - gft
    # - textiel
    # - kerstbomen
    # - kga
    # - pmd
    - papier
  postcode: *REMOVED* # Glimmen,Groningen,Groningen,Nederland
  streetnumber: 2
  upcomingsensor: 1
  printwastetypes: 0
  dateformat: '%d-%m-%Y'
  dateonly: 0
  name: "Afval"
  nameprefix: 1
  builtinicons: 0
  dutch: 1
```
Am I doing something wrong here, has an API changed or did the latest release cause this?

Kind regards, 
Marcel

**Comment by pippyn** [comment]
It looks like the MijnAfvalwijzer API is down at the moment. I'll monitor the status.

**Comment by xirixiz** [comment]
https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer/pull/346 probably fixed this.

**Comment by pippyn** [comment]
Yes it does. Fix released in v5.0.7

Thanks again @xirixiz

**Comment by mdehaas** [comment]
Indeed the issue is fixed. Thanks all!!!!
